ios/Pangea/Info.plist

```diff
@@ -21,11 +21,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0.5</string>
+	<string>1.0.6</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>6</string>
+	<string>11</string>
```

Comment:
Is 11 a correct number? I see previous one is 6, so not sure

Reply:
Yes, we haven't update that in a while and I create several packages before this one

Reply:
Got it